mailgun/route_handlers.py

```diff
@@ -64,11 +84,7 @@ def handle_mailing_list_email_route(request):
                                u"for %s, but we've already handled that.  "
                                u'Skipping.', recipient, message_id)
                 continue
-
-        try:
-            _handle_recipient(request, recipient)
-        except JsonResponse as error_response:
```

Comment:
I was thinking more in terms of the response we're returning might have been more specific.  But I checked, and the one place we raise it, it's just a 500.  So scratch this.